tests/models/aria/test_modeling_aria.py

```diff
@@ -287,10 +287,18 @@ def test_generate_from_inputs_embeds_0_greedy(self):
     def test_generate_from_inputs_embeds_1_beam_search(self):
         pass
 
-    @unittest.skip(reason="Unsupported")
+    @unittest.skip(reason="Dynamic control flow due to MoE")
```

Comment:
I've tagged Aria as not supporting the static cache a few days ago -- I think we can delete these compilation-related skips 🤗